src/app/store/actions/VehicleTestResultModel.actions.ts

```diff
@@ -1,28 +1,59 @@
 import { Action } from '@ngrx/store';
 import { TestResultModel } from '../../models/test-result.model';
+import { Preparer } from '@app/models/preparer';
+import { VIEW_STATE } from '@app/app.enums';
+import { TestStation } from '@app/models/test-station';
 
 export enum EVehicleTestResultModelActions {
   GetVehicleTestResultModel = '[TestResultModel] Get TestResultModel',
   GetVehicleTestResultModelSuccess = '[TestResultModel] Get TestResultModel Success',
   GetVehicleTestResultModelFailure = '[TestResultModel] Get TestResultModel Failure',
+  SetPreparers = '[TestResultComponent] Set Preparers',
```

Comment:
We should split system reference data into separate state. Hence we should have separate action and reducer file. The actions should be LoadPreparers, LoadPreparesSuccess, LoadTestStations, LoadTestStationsSuccess. I will share more light when we discuss